todoapp-main/src/main/java/ru/todoapp/configuration/KafkaConfig.java

```diff
@@ -67,13 +65,43 @@ public NewTopic requestResultTopic() {
     /**
      * Kafka topic creation for registration
      *
-     * @see RegisterRequestDTO
+     * @see ru.todoapp.model.dto.RegisterRequestDTO
      */
     @Bean
     public NewTopic userRegistrationTopic() {
         return new NewTopic(KafkaTopics.REGISTRATION_TOPIC, 1, (short) 1);
     }
 
+    /**
+     * Kafka topic creation for adding tasks
+     *
+     * @see ru.todoapp.model.dto.AddTaskRequestDTO
+     */
+    @Bean
+    public NewTopic addTaskTopic() {
+        return new NewTopic(KafkaTopics.ADD_TASK_TOPIC, 1, (short) 1);
+    }
+
+    /**
+     * Kafka topic creation for fetching tasks request
+     *
+     * @see ru.todoapp.model.dto.FetchTasksRequestDTO
+     */
+    @Bean
+    public NewTopic fetchTasksRequestTopic() {
+        return new NewTopic(KafkaTopics.FETCH_REQUEST_TASKS_TOPIC, 1, (short) 1);
+    }
+
+    /**
+     * Kafka topic creation for fetching tasks response
+     *
+     * @see ru.todoapp.model.dto.FetchTasksResponseDTO
+     */
+    @Bean
+    public NewTopic fetchTasksResponseTopic() {
+        return new NewTopic(KafkaTopics.FETCH_RESPONSE_TASKS_TOPIC, 1, (short) 1);
+    }
+
```

Comment:
I think we should use a constants instead of 1 in this whole class. Probably I just forgot to replace here this numbers with constants.

Reply:
Ok will change